### ISSUE TITLE
main page example is not working

### ISSUE THREAD

**Issue by edmundlaugasson** [AppImage/AppImageUpdate#157]
Currently at [main GitHub page of AppImageUpdate](https://github.com/AppImage/AppImageUpdate) the example with Subsurface does not work as it gives an error and does not update the Subsurface (tested with version 4.9.4 of Subsurface).
![image](https://user-images.githubusercontent.com/6355082/97084651-3db5f980-1621-11eb-914a-e11a281ed4c6.png)
The question is, whether AppImageUpdate requires continuous build or does work with any AppImage file, that has update information?

**Comment by probonopd** [comment]
Indeed, their current build comes without embedded update information:

```
user@FreeBSD$ ./Subsurface-4.9.4-x86_64.AppImage  --appimage-updateinformation
 # (nothing)
```

**Comment by probonopd** [comment]
@TheAssassin do you think we should report this upstream, or just use a different example in our README?

**Comment by TheAssassin** [comment]
Both, maybe? Relying on a single example is not a good idea. And having that be fixed might also help.

Make sure to propose them to manually set update information, so they don't have to rely on guesswork. To some extent, we rely on their AppImage to work correctly.

**Comment by Morganlej** [comment]
As a very first step just remove the example or link here... ASAP! I spent many minutes trying to figure it out.

Why not use your own program as example - have previous version too to download?
As extra benefit there is a fallback version to get.

And thank you for a nice program!  I now added it to Mageia Wiki at https://wiki.mageia.org/en/Ways_to_install_programs#AppImage

**Comment by probonopd** [comment]
Agree. We should not publish a non-working example.
